packages/repo/src/util.ts

```diff
@@ -17,6 +20,25 @@ export const ucanForOperation = async (
   return auth.encodeUcan(ucanForOp)
 }
 
+export const readCar = async (
+  bytes: Uint8Array,
+): Promise<{ root: CID; blocks: BlockMap }> => {
+  const car = await CarReader.fromBytes(bytes)
+  const roots = await car.getRoots()
+  if (roots.length !== 1) {
+    throw new Error(`Expected one root, got ${roots.length}`)
+  }
+  const root = roots[0]
+  const blocks = new BlockMap()
+  for await (const block of car.blocks()) {
```

Comment:
I don't think this was needed 👍 
```suggestion
  for (const block of car.blocks()) {
```

Reply:
hah opposite actually. needed it inside the for loop as well 👌